packages/react/src/runtime/sheet.tsx

```diff
@@ -28,7 +28,15 @@ export const styleBucketOrdering: Bucket[] = [
 /**
  * Holds all style buckets in memory that have been added to the head.
  */
-const styleBucketsInHead: Partial<Record<Bucket, HTMLStyleElement>> = {};
+let styleBucketsInHead: Partial<Record<Bucket, HTMLStyleElement>>;
```

Comment:
Can we remove the `Partial` type?

Reply:
We cannot, when assigning the default value later we get this error:
```
Type '{}' is missing the following properties from type 'Record<Bucket, HTMLStyleElement>': "", l, v, w, and 5 more.ts(2740)
```